catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/CatalogFrameworkImpl.java

```diff
@@ -1004,34 +1002,34 @@ public CreateResponse create(CreateRequest createRequest)
                 }
                 metacard.setAttribute(new AttributeImpl(Metacard.SECURITY, itemPolicyMap));
             }
-            createReq.getProperties()
+            createRequest.getProperties()
                     .put(PolicyPlugin.OPERATION_SECURITY, requestPolicyMap);
 
             for (AccessPlugin plugin : frameworkProperties.getAccessPlugins()) {
-                createReq = plugin.processPreCreate(createReq);
+                createRequest = plugin.processPreCreate(createRequest);
```

Comment:
It is assumed that the request will be changed within the catalog framework. Updating the parameter is not only fine, it is expected.